src/components/homePage/Speakers.vue

```diff
@@ -1,13 +1,26 @@
 <template>
 <div id="speakers">
 	<h2>CUTC 2020</h2>
-	<!-- TAB LAYOUT -->
+	<div id="TABS">
+		<p id="tab" @click="currentPage = 0"> Speakers </p>
+		<p id="tab" @click="currentPage = 1"> Events </p>
+	</div>
+	<div id="content">
+		<p v-if="currentPage == 0"> speaker info </p>
+		<p v-else> event info </p>
+	</div>
```

Comment:
Why is speakers being edited in this PR? Please check which files have been modified before committing :)

Reply:
just noticed that which was weird since i did git status and that file wasn't there (only added Testimonials.vue and testimonials.json) i think it must've happened when i rebased before commiting

Reply:
Could you remove it? If not it's not a big deal but it would be cleaner